SYNTAX.md

```diff
@@ -26,6 +27,7 @@ plan:
 ### Benchmark main properties
 
 - `threads`: Number of threads running your plan concurrently. (Optional, default: 1)
+- `concurrency`: Number of concurrent requests. (Optional, default: 1)
```

Comment:
I am not sure `concurrency` is the right term. `ab` uses `concurrency` where as `wrk` uses `connections`.

Reply:
I like it. :ok_hand: